src/main/java/com/pmm/budgetcalculator/expensing/entity/dto/ExpensePlanDTO.java

```diff
@@ -0,0 +1,19 @@
+package com.pmm.budgetcalculator.expensing.entity.dto;
+
+import com.pmm.budgetcalculator.expensing.entity.ExpenseCategory;
+import lombok.Data;
+
+import java.math.BigDecimal;
+import java.util.Date;
+
+@Data
+public class ExpensePlanDTO {
```

Comment:
Sometimes there is data, sometimes data, noargs, builder, allargs... So what should it be?